theme/blog-btn.scss

```
@@ -2,8 +2,17 @@
   font-size: 14px;
   align-self: center;
   position: relative;
-  display: flex;
-  align-items: center;
+  // Keep the announcement to a single line: a long blog title (the badge
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)